P5/Source/Guidelines/en/SA-LinkingSegmentationAlignment.xml

```diff
@@ -887,9 +887,11 @@ which is either an <rs>XPATH</rs> as defined above or an
 <rs>IDREF</rs>, the value of an <att>xml:id</att>
 occurring in the document addressed by the base URI in effect
 for the pointer.</p>
-<p>Example: the pointer <code>#left(//gap[1])</code> 
+<p>Example: the pointer <code>#left(//supplied)</code>
 indicates the point between the first <code>lb</code> and the first
-  <code>gap</code> in the <ref target="#SATSXP-ex">example</ref> above.</p>
+<code>supplied</code> in the <ref target="#SATSXP-ex">example</ref> above.</p>
```

Comment:
Problem that is not part of this PR: This entire section is poorly encoded (e.g., the “supplied” in this line is encoded as `<code>` rather than `<gi>`, the ID “SATSXP-ex” is attached to an anchor, not to the example, etc.). I am about to open another ticket to list (some of) these problems. Would you, @bwbohl, prefer I mark the new ticket as blocked by this one (thus avoiding conflict worries), or would you prefer to trust git to make merging relatively painless?